docs/PR_562_CI_CHECKLIST.md

```
@@ -0,0 +1,137 @@
+### 2. CI Will Run New Tests
+
+**Current CI config:** `.github/workflows/ci.yml` line ~918
+
+**Check:** CI uses `-only-testing:PulsePlateTests/BMIServiceTests` (legacy), but new tests are:
+- `HTTPClientTests`
+- `APIClientTests`
+- `BMIServiceThinAdapterTests`
+
+**Options:**
+
+**A) CI runs all tests in PulsePlateTests target (default behavior)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)